src/Controller/ConsumptionController.php

```diff
@@ -23,41 +24,44 @@ class ConsumptionController extends AbstractController
      * @param PaginatorInterface $paginator
      * @return array
      */
-    public function index(Request $request, PaginatorInterface $paginator): array
+    public function index(Request $request, PaginatorInterface $paginator)
     {
-        $consumptions = $this->getDoctrine()->getRepository(Consumption::class)->findBy([
-            'user' => $this->getUser()
-        ], [
-            'dateTime' => 'DESC'
-        ]);
+        $templateData = $this->getTemplateData($request, $paginator);
 
-        $consumptions = $paginator->paginate(
-            $consumptions,
-            $request->query->getInt('page', 1),
-            5
-        );
+        if($request->get('pdf')) {
+            $html = $this->renderView('consumption/pdf/default.html.twig', $templateData);
+            $dompdf = new Dompdf();
+            $dompdf->loadHtml($html);
+            $dompdf->setPaper('A4', 'landscape');
+            $dompdf->render();
+            $dompdf->stream('overzicht.pdf', ['Attachment' => true]);
+            exit;
+        }
 
-        return [
-            'consumptions' => $consumptions,
-        ];
+        return $templateData;
     }
 
     /**
      * @Route("/ingenomen-consumpties", methods={"GET"})
      * @Template()
+     * @param Request $request
+     * @param PaginatorInterface $paginator
+     * @return array
      */
-    public function taken(): array
+    public function taken(Request $request, PaginatorInterface $paginator): array
     {
-        $consumptions = $this->getDoctrine()->getRepository(Consumption::class)->findBy([
-            'user' => $this->getUser(),
-            'taken' => 1,
-        ], [
-            'dateTime' => 'DESC'
-        ]);
-
-        return [
-            'consumptions' => $consumptions,
-        ];
+        return $this->getTemplateData($request, $paginator, 1);
+    }
+    /**
+     * @Route("/vergeten-consumpties", methods={"GET"})
+     * @Template()
+     * @param Request $request
+     * @param PaginatorInterface $paginator
+     * @return array
+     */
+    public function forgotten(Request $request, PaginatorInterface $paginator): array
+    {
+        return $this->getTemplateData($request, $paginator,0);
```

Comment:
3e parameter is een boolean, dus ook een boolean doorgeven (true/false ipv 0/1)

Reply:
in de database zijn de values van taken 0 en 1. Moet ik dan int taken gebruiken ipv bool?